### PR TITLE
week2 - Divide and Conquer

### PR DESCRIPTION
**Pull request by dbtkd1711**

### CODE REVIEW

File: dbtkd1711/baekjoon/분할/1074-Z.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    cin >> c;
+
+    int answer = when_visited(r, c, N);
+
+    cout << answer << endl;
+    return 0;
+}
+
+int when_visited(int row, int column, int length) {
+    if(length == 1) {
```
Comment (JaechanAn) [inline dbtkd1711/baekjoon/분할/1074-Z.cpp:22]: return row * 2 + col;

File: dbtkd1711/baekjoon/분할/11729-tower_of_hanoi.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    move_tower(from, by, to, number-1);
+    vector <int> process;
+    process.push_back(from);
+    process.push_back(to);
+    processes.push_back(process);
+    move_tower(by, to, from, number-1);
+
+    return;
+}
+// void move_tower(int from, int to, int number) {
```
Comment (sejin6) [inline dbtkd1711/baekjoon/분할/11729-tower_of_hanoi.cpp:43]: delete unnecessary comment

File: dbtkd1711/baekjoon/분할/1074-Z.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#include <cmath>
+
+using namespace std;
+
+int N, r, c;
+
+int when_visited(int row, int column, int length);
+
+int main() {
+    cin >> N;
```
Comment (JaechanAn) [inline dbtkd1711/baekjoon/분할/1074-Z.cpp:11]: cin >> N >> r  >> c;

File: dbtkd1711/baekjoon/분할/11729-tower_of_hanoi.cpp [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            printf("%d ", processes[i][j]);
+        }
+        printf("\n");
+    }
+    return 0;
+}
+
+void move_tower(int from, int to, int by, int number) {
+    if(number == 1) {
+        vector <int> process;
```
Comment (JaechanAn) [inline dbtkd1711/baekjoon/분할/11729-tower_of_hanoi.cpp:28]: Remove redundant codes

**Pull request merged**